plugin-server/src/main/ingestion-queues/session-recording/blob-ingester/session-offset-high-water-mark.ts

```diff
@@ -22,6 +22,23 @@ const offsetHighWaterMarkKey = (prefix: string, tp: TopicPartition) => {
  */
 export class SessionOffsetHighWaterMark {
     private topicPartitionWaterMarks: Record<string, Record<string, number>> = {}
+
+    getWatermarkFor(topic: string, partition: number): Record<string, number> {
```

Comment:
nit: Why not
```suggestion
    getWatermarkFor(tp: TopicPartition): Record<string, number> {
```

and in the other place?

Reply:
I nearly did and then thought "I hand you the cash not my wallet" 🤣